src/libraries/Common/tests/System/Net/Http/Http3LoopbackServer.cs

```diff
@@ -82,14 +82,14 @@ public override async Task<GenericLoopbackConnection> EstablishGenericConnection
 
         public override async Task AcceptConnectionAsync(Func<GenericLoopbackConnection, Task> funcAsync)
         {
-            using Http3LoopbackConnection con = await EstablishHttp3ConnectionAsync().ConfigureAwait(false);
+            await using Http3LoopbackConnection con = await EstablishHttp3ConnectionAsync().ConfigureAwait(false);
             await funcAsync(con).ConfigureAwait(false);
             await con.ShutdownAsync();
         }
 
         public override async Task<HttpRequestData> HandleRequestAsync(HttpStatusCode statusCode = HttpStatusCode.OK, IList<HttpHeaderData> headers = null, string content = "")
         {
-            using var con = (Http3LoopbackConnection)await EstablishGenericConnectionAsync().ConfigureAwait(false);
+            await using var con = (Http3LoopbackConnection)await EstablishGenericConnectionAsync().ConfigureAwait(false);
```

Comment:
nit: `Http3LoopbackConnection con` instead of `var`

Reply:
It was like this before, I just added the `await`. If we want to address `var` vs. concrete type, we can do that as a separate clean up PR afterwards, wdyt?

Reply:
Changing this particular case, but I'm not digging for all the other occurrences in this PR.